.github/workflows/release.yaml

```diff
@@ -0,0 +1,53 @@
+name: Release
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: open-turo/actions-jvm/lint@v1
+        with:
+          checkout-repo: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  test:
+    name: Test
```

Comment:
Would it be possible to use https://github.com/open-turo/actions-jvm/tree/main/test instead? I think you would just have to have a `.sdkman` file in the repo and keep the step that installs android still